### DISCUSSION TITLE
Is the whole model conditioned after training images?

### DISCUSSION THREAD

**Discussion by dvelm** [TheLastBen/fast-stable-diffusion#893] [Q&A]
I have noticed that also if I don't write the chosen name of subjects in trained images, the subject comes out. Sometimes I write a prompt in the original model and the result is completely different, it's precisely what I write, while in the trained model the output comes with the subject of trained images, also if not written in the prompt.
Is there any way to fix this? Disabling text encoder? Using lower percentage? More steps to train?

**Comment by TheLastBen** [comment]
that's called overfitting, it happens when you train a model on a small dataset, reducing the text encoder to 10% helps with that issue.

**Reply by dvelm** [reply]
thank you, I used 20 images, probably not enough. I try to reduce the text econder and see if the things are better

**Reply by TheLastBen** [reply]
20 is good enough, set the steps to 6000-7000 and the textenc to 12%

**Reply by dvelm** [reply]
I tried 35 images and 7000 step, one time 20 text encoder and the other 12 but also if there is no mention of the trained subject in the prompt, still appears in the image every time there is reference to a person, but if I use the negative prompt of the subject doesn't appear.
I have unchecked fp16 in colab. Could it improve the output if checked? I am trying it
